build/Common.prod.props

```diff
@@ -13,13 +13,15 @@
     <PackageTags>Observability;OpenTelemetry;Monitoring;Telemetry;Tracing;Metrics;Logging</PackageTags>
     <PackageIcon>opentelemetry-icon-color.png</PackageIcon>
     <PackageProjectUrl>https://opentelemetry.io</PackageProjectUrl>
-    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <Authors>OpenTelemetry Authors</Authors>
     <Copyright>Copyright The OpenTelemetry Authors</Copyright>
-    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <PackageOutputPath Condition="$(Build_ArtifactStagingDirectory) != ''">$(Build_ArtifactStagingDirectory)</PackageOutputPath>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+    <PackagePrimaryLicenseFile>$(RepoRoot)\LICENSE.txt</PackagePrimaryLicenseFile>
+    <PackageThirdPartyNoticesFile>$(RepoRoot)\THIRD-PARTY-NOTICES.txt</PackageThirdPartyNoticesFile>
```

Comment:
Do we include this in all packages or only the packages that actually have 3rd party code?

Reply:
The current design is to include it in all packages. Overkill, but much simpler that way! dotnet\runtime does the same. For example here is `System.Diagnostics.DiagnosticSource`:

https://nuget.info/packages/System.Diagnostics.DiagnosticSource/8.0.0

It has the main repo `THIRD-PARTY-NOTICES.txt` which includes license notices for things like ASP.NET, mono, Zlib, etc. Doubt any of that is really in DS.

Reply:
Got it, thanks for the context 👍